packages/containers/src/ComponentForm/fields/MultiSelect/MultiSelect.component.js

```diff
@@ -11,7 +11,7 @@ import callTrigger from '@talend/react-forms/lib/UIForm/trigger';
 export default class MultiSelectField extends React.Component {
 	constructor(props) {
 		super(props);
-		this.state = { selected: props.value };
```

Comment:
`selected` state is not used, `this.props.value` is used instead, and passed to the underlying component